x-pack/legacy/plugins/apm/server/lib/transactions/breakdown/index.test.ts

```diff
@@ -149,7 +193,18 @@ describe('getTransactionBreakdown', () => {
           get: () => 'myIndex' as any,
```

Comment:
Is this still needed?

Reply:
it's needed to satisfy typescript for the shape of the `config` object